toolbox/db/db_get.m

```diff
@@ -218,32 +218,58 @@
         else
             varargout{1} = sAnatFiles;
         end
-
-    % Usage: sFiles = db_get('FunctionalFile', FileType, FileIDs)
-    % Usage: sFiles = db_get('FunctionalFile', FileType, FileNames)
+        
+%% ==== FUNCTIONAL FILE ====
+    % Usage: [sFiles, sItems] = db_get('FunctionalFile', FileIDs,   Fields)
+    % Usage: [sFiles, sItems] = db_get('FunctionalFile', FileNames, Fields)
+    % Usage: [sFiles, sItems] = db_get('FunctionalFile', CondQuery, Fields)
     case 'FunctionalFile'
-        type = args{1};
-        iFiles = args{2};
+        % Parse inputs
+        iFiles = args{1};
+        fields = '*';                              
+        templateStruct = db_template('FunctionalFile');
+
         if ischar(iFiles)
             iFiles = {iFiles};
+        elseif isstruct(iFiles)
+            condQuery = CondQuery;           
```

Comment:
Error: Unrecognized function or variable 'CondQuery'

Reply:
Solved in daed99c